### PR TITLE
Add AtomsAPI service and fetch/Create atom actions

### PR DESCRIPTION
**Pull request by ShaunYearStrong**
Adds some reducer tests

### CODE REVIEW

File: public/js/actions/getAtom.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  console.error(error);
+  return {
+    type:       'SHOW_ERROR',
+    message:    'Could not get atom',
+    error:      error,
+    receivedAt: Date.now()
+  };
+}
+
+export function getAtom(id) {
```
Comment (philmcmahon) [inline public/js/actions/getAtom.js]: you'll need the atomType as well as the id

File: public/js/services/AtomsApi.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { pandaReqwest } from './pandaReqwest';
+import { getStore } from '../util/storeAccessor';
+
+export default {
+
+  fetchAtom: (atomId) => {
+    return pandaReqwest({
+      url: '/api/preview/' + atomId,
```
Comment (philmcmahon) [inline public/js/services/AtomsApi.js]: /api/preview/atomType/id

File: package.json [hunk 3]
```diff
@@ -39,16 +39,19 @@
...
     "redux": "^3.6.0",
     "redux-thunk": "^2.2.0",
+    "reqwest": "^2.0.5",
     "scribe": "guardian/scribe#fcf4e16",
+    "scribe-plugin-keyboard-shortcuts": "^0.1.1",
     "scribe-plugin-link-prompt-command": "^1.0.0",
     "scribe-plugin-sanitizer": "^0.1.10",
-    "scribe-plugin-toolbar": "^1.0.0",
-    "scribe-plugin-keyboard-shortcuts": "^0.1.1"
+    "scribe-plugin-toolbar": "^1.0.0"
```
Comment (clloyd) [inline package.json]: Probably shouldn't have the scribe stuff in this PR?

File: public/js/actions/AtomActions/getAtom.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    type:       'ATOM_GET_REQUEST',
+    id:         id,
+    atomType:   atomType,
+    receivedAt: Date.now()
+  };
+}
+
+function receiveAtom(atom) {
+  return {
+    type:       'VIDEO_GET_RECEIVE',
```
Comment (philmcmahon) [inline public/js/actions/AtomActions/getAtom.js]: video?

Comment (ShaunYearStrong) [inline public/js/actions/AtomActions/getAtom.js]: Ooops!

**Pull request merged**